### PR TITLE
Go api announcement

### PR DESCRIPTION
**Pull request by aturley**
This is the Go API announcement blog post.

    Audience: Go programmers who are interested in creating streaming data
    applications. They are most likely not familiar with Wallaroo, but
    have been involved in building applications that handle events and
    have spent time thinking about issues of scale.
    
    Why they will read it: They want to learn more about the options that
    are available to them for writing scalable applications.
    
    What they will learn from it: They will learn about Wallaroo's core
    concepts and how to build applications in Go with those concepts.
    
    What they will do after they have read it: They will clone the git
    repo and try out some of the examples.

### CODE REVIEW

File: content/post/golang-api.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+    "wallaroo",
+    "golang",
+    "api"
+]
+categories = [
+    "Go API"
+]
++++
+
+We've been working on our processing engine, [Wallaroo](https://github.com/wallaroolabs/wallaroo/tree/release) for just under two years now. Our goal has been to make it as easy to build fast, scale-independent applications for processing data. When we open sourced Wallaroo last year we provided an API that let developers create applications using [Python](python-api.md). Over the last few months we've been working to make Wallaroo available to a wider range of programmers by adding a Go API, and that's what I'd like to talk about today.
```
Comment (SeanTAllen) [inline content/post/golang-api.md]: Python link is broken.

Should we link to `release` or `master` for Wallaroo? I believe previously we were linking to `master`.

File: content/post/golang-api.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+This example uses a TCP sink, but Wallaroo also supports Kafka sinks. Other types of sinks will be added in the future.
+
+#### Serialization and Deserialization
+
+Wallaroo needs to be able to serialize and deserialize objects in order to store them to disk for resiliency and recovery, and also to send them to other worker nodes when the application is being used in a multi-worker cluster. The developer must create code to do this. We've omitted that code in this blog post, but you can learn more about it in the ["Interworker Serialization and Resilience"](https://docs.wallaroolabs.com/book/go/api/interworker-serialization-and-resilience.html) section of our documentation.
+
+## A Scalable Event Processing Application
+
+This application can run on one worker and can scale horizontally by adding more and more workers. Wallaroo's flexibility makes it easy to adapt to whatever partitioning strategy your application requires. Take a look at our documentation for [information about how to run a Wallaroo cluster](https://docs.wallaroolabs.com/book/appendix/clustering.html).
```
Comment (SeanTAllen) [inline content/post/golang-api.md]: clustering link is broken.

File: content/post/golang-api.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+
+## Check It Out
+
+If you're interested in running this application yourself, take a look at the [Wallaroo documentation](https://docs.wallaroolabs.com) and the [word count example application](https://github.com/WallarooLabs/wallaroo/tree/release/examples/go/word_count) that we've built. You'll find instructions on setting up Wallaroo and running applications. And take a look at our [community page](https://www.wallaroolabs.com/community) to sign up for our mailing list or join our IRC channel to ask any question you may have.
+
+You can also watch this video to see Wallaroo in action. Our VP of Engineering walks you through the concepts that were covered in this blog post and then shows the word count application scaling by adding new workers to the cluster.
```
Comment (SeanTAllen) [inline content/post/golang-api.md]: Should the video note that its using the old Python API but the core concepts still hold?

Comment (aturley) [reply content/post/golang-api.md]: I'll change the wording to make that clear.

**Pull request merged**